sdk/js/src/token_bridge/createWrapped.ts

```diff
@@ -70,3 +72,32 @@ export async function createWrappedOnAlgorand(
     senderAddr
   );
 }
+
+export async function createWrappedOnNear(
+  client: nearAccount,
+  tokenBridge: string,
+  attestVAA: Uint8Array
+): Promise<string> {
+  // Could we just pass in the vaa already as hex?
+  let vaa = Buffer.from(attestVAA).toString("hex");
+
+  let res = await client.viewFunction(tokenBridge, "deposit_estimates", {});
+
+  let result = await client.functionCall({
+    contractId: tokenBridge,
+    methodName: "submit_vaa",
+    args: { vaa: vaa },
+    attachedDeposit: new BN(res[1]),
+    gas: new BN("150000000000000"),
+  });
+
+  result = await client.functionCall({
+    contractId: tokenBridge,
+    methodName: "submit_vaa",
+    args: { vaa: vaa },
+    attachedDeposit: new BN(res[1]),
+    gas: new BN("150000000000000"),
+  });
+
+  return nearProviders.getTransactionLastResult(result);
```

Comment:
Does this return like a raw_log or a sequence number?

Reply:
It returns the "account" that holds the new asset